src/payload/blocks/CTA/CTA_3/config.ts

```
@@ -0,0 +1,31 @@
+	},
+	fields: [
+		headerField({
+			defaultValue: "Transform Your Workflow Today",
+		}),
+		subheaderField({
+			defaultValue:
+				"Join thousands of teams already using our platform to build better products faster.",
+		}),
+		callToActionPair({
+			minRows: 0,
+			maxRows: 2,
```

ESLint

[error] Expected "maxRows" to come before "minRows".

(perfectionist/sort-objects)